syft/workers/base.py

```diff
@@ -401,7 +402,13 @@ def send(
 
         return pointer
 
-    def execute_command(self, message: tuple) -> PointerTensor:
+    def execute_command(self, message) -> PointerTensor:
```

Comment:
```suggestion
    def execute_command(self, action: Action) -> PointerTensor:
```
The code below suggests this might be the appropriate type annotation, but I'm not sure if this is right.

Reply:
This will be clearer after your PR is ready